weed/topology/data_center.go

```diff
@@ -30,15 +31,24 @@ func (dc *DataCenter) GetOrCreateRack(rackName string) *Rack {
 	return rack
 }
 
-func (dc *DataCenter) ToMap() interface{} {
-	m := make(map[string]interface{})
-	m["Id"] = dc.Id()
-	var racks []interface{}
+type DataCenterMap struct {
```

Comment:
how about calling this `DataCenterInfo`? 

Same for other `xxxMap` objects.

Reply:
OK, fix in new commit.